logic/src/main/java/nl/jft/logic/LogicConstants.java

```diff
@@ -9,7 +9,7 @@
 
     public static final int INTERNAL_ID = -1;
 
-    public static final LocalDateTime INTERNAL_DATETIME = LocalDateTime.MIN;
+    public static final Date INTERNAL_DATETIME = new Date(Long.MIN_VALUE);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS2386)